hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/GrpcOmTransport.java

```diff
@@ -177,17 +181,22 @@ public void start() throws IOException {
   public OMResponse submitRequest(OMRequest payload) throws IOException {
     OMResponse resp = null;
     boolean tryOtherHost = true;
+    int expectedFailoverCount = 0;
     ResultCodes resultCode = ResultCodes.INTERNAL_ERROR;
     while (tryOtherHost) {
       tryOtherHost = false;
+      expectedFailoverCount = syncFailoverCount.get();
       try {
         resp = clients.get(host.get()).submitRequest(payload);
       } catch (StatusRuntimeException e) {
         if (e.getStatus().getCode() == Status.Code.UNAVAILABLE) {
           resultCode = ResultCodes.TIMEOUT;
         }
         Exception exp = new Exception(e);
-        tryOtherHost = shouldRetry(unwrapException(exp));
+        /*e.getStatus().getCode() == Status.Code.
```

Comment:
Should this be deleted?

Reply:
Yes.  Left-over thought thus commented out.  I'll remove it.  Thanks.